docs/howto/without_active_record.md

```diff
@@ -0,0 +1,91 @@
+# How to use ActiveModelSerializers without ActiveRecord
+
+Every model that is to be serialized must implement the following methods
```

Comment:
I don't think it hurts to have more general-use examples.
But, you're also saying that `ActiveModel::Model` provides the hash attribute instantiate? I did not know that.

@bf4 you should have write access to this repo/branch - I haven't used the lint before, but if you want to add documentation here (or maybe it would be better elsewhere)? I think that could be beneficial.